src/com/company/Main.java

```diff
@@ -20,53 +21,55 @@ public static void main(String[] args) {
 
         isFileFormatSupported(file);
 
-        if(!file.exists() || !isFileFormatSupported(file)) {
+        if (!file.exists() || !isFileFormatSupported(file)) {
             System.out.println("file is not valid");
```

Comment:
Start all the sysout messages with capital letters and end with proper punctuations.

Reply:
done with that.